libmultilabel/common_utils.py

```diff
@@ -156,3 +158,18 @@ def is_multiclass_dataset(dataset, label="label"):
             a multi-class problem."""
         )
     return ratio == 1.0
+
+
+def timer(func):
```

Comment:
I think this part is in [PR#350](https://github.com/ASUS-AICS/LibMultiLabel/pull/350#issue-2086031916)?

Reply:
I used it to record the running time of some functions in AttentionXML. I'll remove it from the PR.

Reply:
Since the new PR has been merged. I now rebased everything on the master branch.